### DISCUSSION TITLE
Rollback to previous revision conflicts with ArgoCD

### DISCUSSION THREAD

**Discussion by mrmartinez95** [argoproj/argo-rollouts#2571] [Q&A]
Hey guys,

I have a Rollout object with a reference to an existing deployment, with a canary strategy, and it's using Istio for traffic routing. 

Every rollout works as expected but I'm getting a conflict between Rollout and ArgoCD when trying to rollback to a previous revision. Rollout modifies the live deployment so ArgoCD syncs back to the deployment's git current version.

Replacing the deployment with a Rollout object is not an option for me. Is there something I can do to workaround this conflict issue?

Thanks!!

**Comment by kostis-codefresh** [comment]
If you don't want to convert to Rollout, only option would be to ignore diffs in the deployment https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/
